tests/test_web/test_api_v1.py

```diff
@@ -578,12 +579,35 @@ def test_add_bundles_graph_update_mode_not_allowed(
     assert rv.status_code == 403
     error_msg = (
         '"graph_update_mode" can only be used on the'
-        ' following index image: [\'some-unique-index\']'
+        f' following index image: {graph_update_mode_list}'
     )
     assert error_msg == rv.json['error']
     mock_smfsc.assert_not_called()
 
 
+@pytest.mark.parametrize('from_index', ('some-common-index:v4.15', 'another-common-index:v4.15'))
```

Comment:
please add also:  'another-common-index:v4.14', but which test should fail in current settings.

Reply:
If I am correct, this is already achieved in the test 'test_add_bundles_graph_update_mode_not_allowed' with 'some-common-index:v4.15' as equivalent to  'another-common-index:v4.14'. (Graph_update mode list contains 'some-unique-index' and  'some-common-index:v1.14'.

Reply:
I was blind. Thanks.